src/Compilers/Lua/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -282,6 +282,16 @@ public static SyntaxToken Token(SyntaxTriviaList leading, SyntaxKind kind, strin
         public static SyntaxToken Literal(string text, double value) =>
             new(InternalSyntax.SyntaxFactory.Literal(ElasticMarker.UnderlyingNode, text, value, ElasticMarker.UnderlyingNode));
 
+        /// <summary>
+        /// Creates a token with kind NumericLiteralToken from the text and corresponding 8-byte floating point value.
+        /// </summary>
+        /// <param name="leading">A list of trivia immediately preceding the token.</param>
+        /// <param name="text">The raw text of the literal.</param>
+        /// <param name="value">The 8-byte unsigned integer value to be represented by the returned token.</param>
+        /// <param name="trailing">A list of trivia immediately following the token.</param>
+        public static SyntaxToken Literal(SyntaxTriviaList leading, string text, ulong value, SyntaxTriviaList trailing) =>
+            new(InternalSyntax.SyntaxFactory.Literal(leading.Node, text, value, trailing.Node));
+
```

Comment:
Would be nice to have the other convenience methods here (`Literal(string, ulong)` and `Literal(ulong)`).

The overload that doesn't accept a `string` will need a method in `ObjectDisplay` to have support for formatting ulongs (which should also add the `ULL` prefix).